docs/proposals/assets/multiple-dns-provider-support/azure/azure.md

```diff
@@ -0,0 +1,56 @@
+
+## Azure
+
+https://portal.azure.com/
+
+Azure supports Weighted and Geolocation routing policies, but requires records to alias to a Traffic Manager resource that must also be created in the users acocunt https://learn.microsoft.com/en-us/azure/traffic-manager/traffic-manager-routing-methods
+
+Notes:
+
+* A Traffic Manager Profile is created per record set and is created with a routing method (Weighted or Geographic) https://portal.azure.com/#view/Microsoft_Azure_Network/LoadBalancingHubMenuBlade/~/TrafficManagers
+* Only a singe IP can be added to a DNSRecord set. A traffic manager profile must be created and aliased from a DNSRecord set for anything that involves more than a single target.
```

Comment:
record set is a strange name for something that only has 1 record 🤷

Reply:
pricing seems similar if maybe a bit cheaper for azure (was just curious) https://azure.microsoft.com/en-us/pricing/details/traffic-manager/

Reply:
> record set is a strange name for something that only has 1 record shrug

Yeah, not really sure why this is the case tbh, but you have to use the Traffic Manager profiles to do weighted anyway which we want as a minimum.

![image](https://github.com/Kuadrant/multicluster-gateway-controller/assets/327352/2df149c4-f6e7-44da-91db-b657fbaba22b)

> a bit cheaper for azure (was just curious) https://azure.microsoft.com/en-us/pricing/details/traffic-manager/

I couldn't really work out what the real cost was going to be, even using their calculator, depends on queries made. I was going to keep an eye on the cost of the example geo one i setup and see what the actual cost is. Azure has quite a good screen to break down the costs of each resource.